README.md

```diff
@@ -31,79 +37,3 @@ npm start
 Which will start a server at `http://localhost:8080`, proxying API requests to the replica at port 8000.
 
 Note: while the protocol doesn't need or have a UI, the asset's canister here serves as a simple way of testing the protocol by simulating how an application might create Entities and connect them via Bridges.
-
-Call from CLI (local):
```

Comment:
we can store these CLI calls in a separate file (as I use them regularly). Good thing to delete them from the Readme though

Reply:
including the calls to top up the canisters below